src/services/DiscordService.ts

```diff
@@ -237,10 +238,10 @@ export class DiscordService {
     }
   }
 
-  static finishProposal(id: string, title: string, outcome: string, winnerChoice?: string) {
+  static finishProposal(id: string, title: string, outcome: ProposalStatus, winnerChoice?: string) {
     if (DISCORD_SERVICE_ENABLED) {
       inBackground(async () => {
-        const action = `Proposal has ended with outcome ${outcome}`
+        const action = `Proposal has ended with outcome ${outcome.toUpperCase().replaceAll('_', ' ')}`
```

Comment:
We are also doing this replace in `CategoryPill`. Maybe we could extract all this to a function and reuse it in both files. (with toUpperCase included)